eoscostarica/guide.eoscostarica.io
# Feat(frontend): Pair and Mob Programming 236

### GH Issue

Resolves #236  

### Steps to test
1. Go to Software Development in Pair or Mob
2. Check content is right.


#### CheckList
- [ ] The content is equal  to https://github.com/eoscostarica/guias.eoscostarica.io/pull/216 but in English
